<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdPreferencesDialog</class>
 <widget class="QDialog" name="QxrdPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qxrd Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="detector">
      <attribute name="title">
       <string>Detector</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Detector Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_DetectorTypeCombo"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QStackedWidget" name="m_DetectorPrefsStack">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="m_SimulatedDetectorPrefs">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>60</y>
             <width>231</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Simulated Detector</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="m_PEDetectorPrefs">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>40</y>
             <width>281</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Perkin Elmer Flat Panel Detector</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="m_PilatusDetectorPrefs">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>211</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Pilatus Pixel Array Detector</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>60</y>
             <width>251</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Not yet implemented</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="m_EpicsDetectorPrefs">
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>211</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Epics Area Detector</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>60</y>
             <width>281</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Not yet implemented</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Processing</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Use CUDA device if available</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Servers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QCheckBox" name="m_RunSpecServer">
         <property name="text">
          <string>Run 'spec' server socket</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>On Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_SpecServerPort">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_RunSimpleServer">
         <property name="text">
          <string>Run 'simple' server socket</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>On Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_SimpleServerPort">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>134</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="memory">
      <attribute name="title">
       <string>Memory</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Memory to reserve for images in 32 bit systems (MB):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="m_ReservedMemory32"/>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>180</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="m_ReservedMemory64"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Memory to reserve for images in 64 bit systems (MB):</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="debug">
      <attribute name="title">
       <string>Debug</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Debug Level</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="m_DebugLevelSpinBox"/>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>289</width>
           <height>180</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QxrdPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QxrdPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
